ci-operator/jobs/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-community-4.9-presubmits.yaml

```diff
@@ -0,0 +1,265 @@
+presubmits:
+  openshift/windows-machine-config-operator:
+  - agent: kubernetes
+    always_run: true
```

Comment:
Consider introducing https://github.com/openshift/release/pull/23899/commits/dade17520eeb7f2eb96341d5b050b4786332ae69 as part of this commit..